libs/importData.js

```diff
@@ -189,7 +189,17 @@ const importData = async function(cozyClient, data, options) {
 
   for (let doctype of Object.keys(data)) {
     let docs = data[doctype]
-    assert(docs, 'No documents for doctype ' + doctype)
+
+    try {
+      const hasDocs = docs && docs.length && docs.length > 0
+      assert(hasDocs, 'No documents for doctype ' + doctype)
+    } catch (err) {
+      console.warn(err)
+      continue
+    }
```

Comment:
Why not a simple `if` ?

Reply:
No particular reason. But it's simpler with a `if`, you're right.

Reply:
Cool :)